tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -364,12 +365,6 @@ public function test_attach_exchange_info_to_order_with_different_order_currency
 		$this->assertEquals( 0.853, $order->get_meta( '_wcpay_multi_currency_stripe_exchange_rate' ) );
 	}
 
-	public function test_payment_fields_outputs_fields() {
-		$this->wcpay_gateway->payment_fields();
-
-		$this->expectOutputRegex( '/<div id="wcpay-card-element"><\/div>/' );
```

Comment:
No `wcpay-card-element` is rendered anymore. Instead, [the UPE gateway test tests the correct input element.](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/unit/payment-methods/test-class-upe-payment-gateway.php#L371)